.github/scripts/update_tools.py

```diff
@@ -43,9 +38,8 @@ def get_latest_release(package_name, package_info):
 
 def update_versions(file_path):
     """Update the tools versions in the specified file while preserving format."""
-    data = load_packages()
     updates_made = False
-    packages = data.get("packages", {})
+    data,packages = load_packages()
```

Comment:
Either you give the `file_path` here or you remove the args file_path, it's weird to use the one provided in argument in some place and use the const in some other

To me you could remove the arg and only keep the const